app/src/main/res/values/strings.xml

```diff
@@ -78,14 +78,18 @@
     <string name="decryption_in_progress">Decrypting database</string>
     <string name="decrypt_fail_current_key_null">Unable to decrypt database as current key is null</string>
 
+    <!-- Image -->
+    <string name="add_image">Add picture</string>
+    <string name="remove_image">Remove</string>
+    <string name="change_image">Change</string>
+    <string name="image_description">This is a user selected image</string>
+
     <!--- counter creation -->
-    <string name="create_counter">Create Counter</string>
-    <string name="create_counter_name">What is the name of your counter?</string>
-    <string name="create_counter_start_date">When did you begin?</string>
-    <string name="create_counter_reason">Why are you doing this?</string>
+    <string name="create_counter">Add Tracker</string>
     <string name="placeholder_counter_name">Name</string>
-    <string name="placeholder_counter_reason">Reason (Optional)</string>
-    <string name="placeholder_date">dd/mm/yyyy</string>
+    <string name="placeholder_counter_reason">Reason %1$d (optional)</string>
+    <string name="placeholder_date"> Date (dd/mm/yyyy)</string>
```

Comment:
Can you type in a date without opening the mat3 datepicker still? if not remove the format from this string i think

Reply:
i believe i still need the Date (dd/mm/yyyy) as it acts as the text boxes text whilst a date isnt selected, unless this isnt what your talking about?

Reply:
removed